networking/ipv6-control-plane.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Configure Kubernetes control plane to operate over IPv6
+description: Configure the Kubernetes control plane to operate over IPv6 for dual stack or IPv6 only.
+canonical_url: '/networking/ipv6-control-plane'
+---
+
+### Big picture
+
+Depending on your networking set up, you may want to configure the Kubernetes control plane to operate over IPv6 for dual stack or IPv6 only.
```

Comment:
Maybe better: "If you have IPv6 connectivity between your nodes and workloads, you may also want to configure the Kubernetes control plane to operate over IPv6, instead of IPv4."  WDYT?

Reply:
Better. Fixed.